examples/response-metrics/response_metrics.conf

```diff
@@ -0,0 +1,24 @@
+location = /report_metric {
+  internal;
+  resolver 8.8.8.8;
+
+  set $user_key $arg_user_key;
+  set $metric_name $arg_metric_name;
+  set $document_count $arg_count;
+  set $user_credentials transactions[0][user_key]=$user_key;
+  set $metrics transactions[0][usage][$metric_name]=$document_count;
+  set $reporting_url https://su1.3scale.net:443;
```

Comment:
We tried using `$backend_endpoint` here, but it's value was `"backend_upstream"` which resulted in a peering lookup error `failed to set current backend peer: missing peers while connecting to upstream`.